README.md

```diff
@@ -282,6 +282,63 @@ When `user1` logs in, he/she will be able to manage `myAppId1` and `myAppId2` fr
 
 When *`user2`*  logs in, he/she will only be able to manage *`myAppId1`* from the dashboard.
 
+## Use Read-Only masterKey
+
+Starting parse-server 2.6.5, it is possible to provide a readOnlyMasterKey to parse-server to prevent mutations on objects from a client.
+If you want to protect your dashboard with this feature, just use the readOnlyMasterKey instead of the masterKey. All write calls will fail.
+
+### Making an app read-only for all users
+
+Start your parse server with
```

Comment:
parse-server for consistency.

Reply:
Yes!

Reply:
also *could* put `readOnlyMasterKey` `masterKey` insteaxc of masterKey....i.e. mark as code.

Reply:
yes
!